cmd/orgs.go

```diff
@@ -145,7 +146,8 @@ func orgsListCmd(ctx *cli.Context) error {
 		fmt.Printf("%s\n", o.Body.Name)
 	}
 
-	hints.Display(hints.PersonalOrg)
+	fmt.Printf("\nYou are a member of (%s) org%s\n",
```

Comment:
No more personal org hint?

Reply:
I removed it because it made the output really messy, would be fine with adding it back.